.gitignore

```diff
@@ -11,3 +11,6 @@ tags
 .python-version
 .kapitan
 .coverage
+hack
```

Comment:
Not sure if these should be added here. At least `ibmcloud` and `hack` seem overly specific.

Reply:
that was my testing, will remove it